app/javascript/history.js

```diff
@@ -1,6 +1,7 @@
 Vue.filter('phone', function (phone) {
-  return phone.replace(/[^0-9]/g, '')
-    .replace(/(\d{3})(\d{3})(\d{4})/, '($1) $2-$3');
+  return phone.replace(.,''); //Temporarily hide phone number until we can re-train phone center staff
```

Comment:
Revert these changes

Reply:
Change the line here instead - https://github.com/CrisisCleanup/crisiscleanup/blob/master/app/javascript/components/history/HistoryVue.vue#L6

Reply:
Specfically remove this part of the line- `, {{claimedByUser.u_mobile | phone}}`